### ISSUE TITLE
Hotkeys not working in MSI GF63 11UC

### ISSUE THREAD

**Issue by soumyaDghosh** [systemd/systemd#26201]
### systemd version the issue has been seen with

systemd 251 (251.4-1ubuntu7)

### Used distribution

Ubuntu 22.10

### Linux kernel version used

5.19.0-29-generic

### CPU architectures issue was seen on

x86_64

### Component

hardware database files

### Expected behaviour you didn't see

Fn+F3 = Disable Touchpad
Fn+F10= Enable/Disable Airplane mode

### Unexpected behaviour you saw

Pressing the above keys does nothing

### Additional program output to the terminal or log subsystem illustrating the issue

**Pressing Fn+F3**
````
- evdev:
    - [ 20, 294833,   4,   4,      30] # EV_MSC / MSC_SCAN                 30 (obfuscated)
    - [ 20, 294833,   1,  29,       1] # EV_KEY / KEY_LEFTCTRL              1
    - [ 20, 294833,   0,   0,       0] # ------------ SYN_REPORT (0) ---------- +16170ms
  - evdev:
    - [ 20, 295921,   4,   4,      30] # EV_MSC / MSC_SCAN                 30 (obfuscated)
    - [ 20, 295921,   1, 125,       1] # EV_KEY / KEY_LEFTMETA              1
    - [ 20, 295921,   0,   0,       0] # ------------ SYN_REPORT (0) ---------- +1ms
  - evdev:
    - [ 20, 306350,   4,   4,      30] # EV_MSC / MSC_SCAN                 30 (obfuscated)
    - [ 20, 306350,   1,  85,       1] # EV_KEY / KEY_ZENKAKUHANKAKU        1
    - [ 20, 306350,   0,   0,       0] # ------------ SYN_REPORT (0) ---------- +11ms
  - evdev:
    - [ 20, 316232,   4,   4,      30] # EV_MSC / MSC_SCAN                 30 (obfuscated)
    - [ 20, 316232,   1,  29,       0] # EV_KEY / KEY_LEFTCTRL              0
    - [ 20, 316232,   0,   0,       0] # ------------ SYN_REPORT (0) ---------- +10ms
  - evdev:
    - [ 20, 337595,   4,   4,      30] # EV_MSC / MSC_SCAN                 30 (obfuscated)
    - [ 20, 337595,   1, 125,       0] # EV_KEY / KEY_LEFTMETA              0
    - [ 20, 337595,   0,   0,       0] # ------------ SYN_REPORT (0) ---------- +21ms
  - evdev:
    - [ 20, 346754,   4,   4,      30] # EV_MSC / MSC_SCAN                 30 (obfuscated)
    - [ 20, 346754,   1,  85,       0] # EV_KEY / KEY_ZENKAKUHANKAKU        0
    - [ 20, 346754,   0,   0,       0] # ------------ SYN_REPORT (0) ---------- +9ms
  # Current time is 22:04:55`


``````
The airplane mode doesn't record

**Comment by soumyatheman** [comment]
Hey anyone please help out.

**Comment by yuwata** [comment]
Please try to modify [60-keyboard.hwdb](https://github.com/systemd/systemd/blob/main/hwdb.d/60-keyboard.hwdb). The file contains some basic instructions.
If you find a way to fix your issue, then please submit the change as a PR. Thank you.

**Comment by soumyatheman** [comment]
> Please try to modify [60-keyboard.hwdb](https://github.com/systemd/systemd/blob/main/hwdb.d/60-keyboard.hwdb). The file contains some basic instructions. If you find a way to fix your issue, then please submit the change as a PR. Thank you.

Please help me with debugging, what is the thing, that I should write? It's talking about hexcode, but how could I get the hexcode?

**Comment by soumyatheman** [comment]
@yuwata Hey, as you said, I tried but it's not working. Also some of keys are not even recognised? What should be next steps? Can you please guide me?

**Comment by yuwata** [comment]
Sorry for late response.

> ```
> evdev:input:b0011v0001p0001*
>  KEYBOARD_KEY_76=touchpad_toggle
> ```

Where the value `76` comes from??

Not sure, but maybe the report #25968 and its fix 45062a62ac6c888327d57c585f1cc2c47a5bfd2c can help you about how to configure the correct setting.

**Comment by soumyaDghosh** [comment]
> Sorry for late response.
> 
> > ```
> > evdev:input:b0011v0001p0001*
> >  KEYBOARD_KEY_76=touchpad_toggle
> > ```
> 
> Where the value `76` comes from??
> 
> Not sure, but maybe the report #25968 and its fix [45062a6](https://github.com/systemd/systemd/commit/45062a62ac6c888327d57c585f1cc2c47a5bfd2c) can help you about how to configure the correct setting.

Don't know but I'm getting different value now.



`````
Event: time 1676130366.907483, type 4 (EV_MSC), code 4 (MSC_SCAN), value 1d
Event: time 1676130366.907483, type 1 (EV_KEY), code 29 (KEY_LEFTCTRL), value 1
Event: time 1676130366.907483, -------------- SYN_REPORT ------------
Event: time 1676130366.909149, type 4 (EV_MSC), code 4 (MSC_SCAN), value db
Event: time 1676130366.909149, type 1 (EV_KEY), code 125 (KEY_LEFTMETA), value 1
Event: time 1676130366.909149, -------------- SYN_REPORT ------------
Event: time 1676130366.910066, type 4 (EV_MSC), code 4 (MSC_SCAN), value 76
Event: time 1676130366.910066, type 1 (EV_KEY), code 85 (KEY_ZENKAKUHANKAKU), value 1
Event: time 1676130366.910066, -------------- SYN_REPORT ------------
Event: time 1676130366.911759, type 4 (EV_MSC), code 4 (MSC_SCAN), value 1d
Event: time 1676130366.911759, type 1 (EV_KEY), code 29 (KEY_LEFTCTRL), value 0
Event: time 1676130366.911759, -------------- SYN_REPORT ------------
Event: time 1676130366.914096, type 4 (EV_MSC), code 4 (MSC_SCAN), value db
Event: time 1676130366.914096, type 1 (EV_KEY), code 125 (KEY_LEFTMETA), value 0
Event: time 1676130366.914096, -------------- SYN_REPORT ------------
Event: time 1676130366.916308, type 4 (EV_MSC), code 4 (MSC_SCAN), value 76
Event: time 1676130366.916308, type 1 (EV_KEY), code 85 (KEY_ZENKAKUHANKAKU), value 0
``````
I also get multi key presses at the same time.

**Comment by poettering** [comment]
Pretty sure this has long been generically addressed via 4e3a50169bef2952544cd6418d9ffd5075202ca2. Closing.
